help/destinations/catalog/crm/hubspot.md

```diff
@@ -0,0 +1,188 @@
+
+#### You must have a [!DNL HubSpot] account {#prerequisites-account}
+
+In order to export data from Platform to your [!DNL Hubspot] account you need to have a [!DNL HubSpot] account. If you do not have one already, visit the [Set up your HubSpot account](https://knowledge.hubspot.com/get-started/set-up-your-account) page and follow the guidance to register and create your account.
+
+#### Gather the [!DNL HubSpot] private app access token {#gather-credentials}
+
+You need your [!DNL HubSpot] **Access token** to allow the [!DNL HubSpot] destination to make API calls through your [!DNL HubSpot] private app within your [!DNL HubSpot] account. The **Access token** serves as the **Bearer token** when you [authenticate the destination](#authenticate).
+
+If you do not have a private app, follow the documentation to [Create a private app in [!DNL HubSpot]](https://developers.hubspot.com/docs/api/private-apps).
+
+>[!IMPORTANT]
+>
+> The private app should be assigned the scopes below:
+> `crm.objects.contacts.write`, `crm.objects.contacts.read`
+> `crm.schemas.contacts.write`, `crm.schemas.contacts.read`
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| `Bearer token` | The `Access token` of your [!DNL HubSpot] private app. <br>To obtain your [!DNL HubSpot] **Access token** follow the [!DNL HubSpot] documentation to [make API calls with your app's access token](https://developers.hubspot.com/docs/api/private-apps#make-api-calls-with-your-app-s-access-token). | `pat-na1-11223344-abcde-12345-9876-1234a1b23456` |
+
+## Guardrails {#guardrails}
+
+[!DNL HubSpot] private apps are subject to [Rate Limits](https://developers.hubspot.com/docs/api/usage-details). The number of calls your private app can make is based on your [!DNL HubSpot] account subscription and whether you've purchased the API add-on. Additionally also refer to the [Other Limits](https://developers.hubspot.com/docs/api/usage-details#other-limits).
+
+## Supported identities {#supported-identities}
+
+[!DNL HubSpot] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Example|Description|Mandatory|
+|---|---|---|---|
+| `email` | `test@test.com` | Email address of the contact.| Yes |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Each segment status in [!DNL HubSpot] gets updated with the corresponding audience status from Platform, based on the **[!UICONTROL Mapping ID]** value provided during the [audience scheduling](#schedule-segment-export-example) step.</li></ul> |
```

Comment:
The use of "segment" terminology is now deprecated. The documentation template page (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en%C3%A2) was also updated with the use of "audience" instead of "segment".

Please update all occurrences of "segment" with "audience" (only in text, not in code samples or API calls).